libsolidity/interface/GasEstimator.cpp

```diff
@@ -142,6 +142,12 @@ GasEstimator::GasConsumption GasEstimator::functionalEstimation(
 			calldata,
```

Comment:
Not strictly related to this PR, just for better documentation: there could be a comment above this block saying that  return `hashValue` for `div(calldataload(0), 1 << (8 * 28))`